### PR TITLE
Hotfix/union snowplow events

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
- Adds new "view" type Snowplow event models. These are temporary views to join events from `2020-07-28` incoming from our old `ft_snowplow` schema (outdated connector), with the new events incoming into our new `ft_snowplow_20200813` schema.
- It updates the `snowplow_base_event` and `snowplow_payload_event` models to reference the new views.

#### What are the relevant tickets?
- https://dosomething.slack.com/archives/C0196AZBB5X/p1597341244000200

### CODE REVIEW

File: quasar/dbt/models/phoenix_events/snowplow_payload_event.sql [hunk 1]
```diff
@@ -13,7 +13,7 @@ SELECT
     payload::jsonb #>> '{contextSource}' AS context_source,
     payload::jsonb #>> '{value}' AS context_value,
     _fivetran_synced AS ft_timestamp
-FROM {{ source('snowplow', 'snowplow_event') }}
+FROM {{ source('ft_snowplow_payloads') }}
```
Comment (blisteringherb) [inline quasar/dbt/models/phoenix_events/snowplow_payload_event.sql:16]: Should this be a `ref` since it's a derived table?

**Comment by blisteringherb** [comment]
Do we need to add incremental build parameters to the new derived tables like we have for the current snowplow tables? It won't be an issue for a while, but should we future proof it now?

File: quasar/dbt/models/phoenix_events/snowplow_payload_event.sql (hunk 1 above)
Comment (rapala61) [reply quasar/dbt/models/phoenix_events/snowplow_payload_event.sql]: Yes, thanks!

**Comment by rapala61** [comment]
@blisteringherb ready for another 👀!

Re: incremental builds. These new event tables are materialized views so they don't take any extra space. The snowplow models that use incremental materialization will now first "execute" the view in memory and grab the events that they need based on the incremental query.

**Comment by rapala61** [comment]
Tested locally. I think everything is setup properly but I'm having some strange hangup when running this locally.

```
pipenv run dbt run --models phoenix_events --profile qa
Running with dbt=0.16.1
Found 54 models, 72 tests, 3 snapshots, 0 analyses, 130 macros, 0 operations, 0 seed files, 28 sources

16:28:28 | Concurrency: 4 threads (target='default')
16:28:28 | 
16:28:28 | 1 of 10 START view model public.ft_snowplow_events_20200813.......... [RUN]
16:28:28 | 2 of 10 START view model public.ft_snowplow_payloads_20200813........ [RUN]
16:28:28 | 3 of 10 START view model public.ft_snowplow_ua_parser_context_20200813 [RUN]
16:28:29 | 3 of 10 OK created view model public.ft_snowplow_ua_parser_context_20200813 [CREATE VIEW in 1.15s]
16:28:29 | 1 of 10 OK created view model public.ft_snowplow_events_20200813..... [CREATE VIEW in 1.16s]
16:28:29 | 2 of 10 OK created view model public.ft_snowplow_payloads_20200813... [CREATE VIEW in 1.16s]
16:28:29 | 4 of 10 START incremental model public.snowplow_base_event........... [RUN]
16:28:29 | 5 of 10 START incremental model public.snowplow_payload_event........ [RUN]
16:28:31 | 5 of 10 OK created incremental model public.snowplow_payload_event... [INSERT 0 3 in 1.45s]
16:28:31 | 4 of 10 OK created incremental model public.snowplow_base_event...... [INSERT 0 1 in 1.95s]
16:28:31 | 6 of 10 START incremental model public.snowplow_raw_events........... [RUN]
16:28:33 | 6 of 10 OK created incremental model public.snowplow_raw_events...... [INSERT 0 4 in 1.37s]
16:28:33 | 7 of 10 START incremental model public.snowplow_phoenix_events....... [RUN]
16:28:34 | 7 of 10 OK created incremental model public.snowplow_phoenix_events.. [INSERT 0 1 in 1.62s]
16:28:34 | 8 of 10 START incremental model public.snowplow_sessions............. [RUN]
16:28:34 | 9 of 10 START incremental model public.phoenix_events_combined....... [RUN]
16:28:35 | 8 of 10 OK created incremental model public.snowplow_sessions........ [INSERT 0 1 in 1.11s]
16:28:35 | 10 of 10 START incremental model public.phoenix_sessions_combined.... [RUN]
16:29:11 | 10 of 10 OK created incremental model public.phoenix_sessions_combined [INSERT 0 2 in 35.73s]
16:36:01 | 9 of 10 OK created incremental model public.phoenix_events_combined.. [INSERT 0 1 in 446.50s]
Database error while running on-run-end
Encountered an error:
Database Error
  server closed the connection unexpectedly
  	This probably means the server terminated abnormally
  	before or while processing the request.
```

**Pull request merged**